src/main/java/Main.java

```diff
@@ -0,0 +1,39 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        StudentGroup studentGroup = new StudentGroup(new Student(1, "Vova", "Lvov"));
+
+        Student leader = studentGroup.changeGroupLeader(new Student(2, "Ivan", "Bohyn"));
```

Comment:
`Student` object could be kept in variables like:
`student1`, `student2` and etc